packages/travix-ui-kit/components/stepper/stepper.js

```diff
@@ -0,0 +1,130 @@
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import React, { Component } from 'react';
+
+/**
+ * General Stepper component
+ */
+class Stepper extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentValue: props.initValue,
+    };
+  }
+
+  componentWillReceiveProps(newProps) {
+    if (newProps.initValue !== this.state.currentValue) {
+      this.setState({
+        currentValue: newProps.initValue,
+      });
+    }
+  }
+
+  setValue = (newValue) => {
+    const { onChange } = this.props;
+
+    this.setState({
+      currentValue: newValue,
+    }, () => {
+      if (onChange) {
+        onChange(newValue);
+      }
+    });
+  }
+
+  tryDecreaseValue = (e) => {
+    e.preventDefault();
+    const newValue = this.state.currentValue - 1;
+
+    if (newValue < this.props.minValue) {
+      return;
+    }
+
+    this.setValue(newValue);
+  }
+
+  tryIncreaseValue = (e) => {
+    e.preventDefault();
+    const newValue = this.state.currentValue + 1;
+
+    if (newValue > this.props.maxValue) {
+      return;
+    }
+
+    this.setValue(newValue);
+  }
+
+  render() {
+    const { inputName, minValue, maxValue } = this.props;
+    const { currentValue } = this.state;
+
+    const buttonDecreaseClasses = classnames({
+      'icon icon-100 icon-Buttons_arrowMinus ui-stepper__button ui-stepper__button-decrease': true,
```

Comment:
Why you use our internal classes in the ui kit?? `icon` `icon-100` `icon-Buttons_arrowMinus`